packages/@uppy/companion/src/server/provider/zoom/index.js

```diff
@@ -185,13 +187,13 @@ class Zoom extends Provider {
   }
 
   _initializeData (body, initialEnd = null) {
-    let end = initialEnd || moment()
-    const accountCreation = adapter.getAccountCreationDate(body)
-    const defaultLimit = end.clone().subtract(DEFAULT_RANGE_MOS, 'months').date(1)
+    let end = initialEnd || moment.utc().tz(body.timezone || 'UTC')
```

Comment:
we convert `cursor` and `end` (and everything that relies on it) to local time. this was done this way (rather than just adjusting display in adapter) because we need to show the information in local time anyways, and we need access to from / to dates in local timezone to correctly filter out extra results in the `/provider/list` call once a user clicks into a date range.   I couldn't think of a better way to do it that would retain this from/to local times